config/crd/bases/req.nephio.org_capacities.yaml

```diff
@@ -41,6 +41,10 @@ spec:
                   throughput
                 pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
                 x-kubernetes-int-or-string: true
+              maxNFConnections:
```

Comment:
@henderiw What does this field do? Is it for NFs SBI interface? Restricting the number of SBI communications a NF can have at the same point of time?

Reply:
This should be made optional --- from my original SMFDeployment, it has an optional field for maxUPFConnections (or something like that) to denote the maximum number of UPFs connected to this SMF. This parameter can potentially have an impact on memory need, hence added (but not required)